tests/macros/InterfaceOverview.test.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Test the API Interface Overview page content macro
+ *
+ * @prettier
+ */
+
+// Get necessary modules
+const {
+    assert,
+    describeMacro,
+    beforeEachMacro,
+    itMacro,
+    readJSONFixture
+} = require('./utils');
+const { JSDOM } = require('jsdom');
+
+const INTERFACE_DATA = JSON.stringify([
+    {
+        XMLHttpRequestEventTarget: {
+            inh: 'EventTarget',
+            impl: []
+        }
+    }
+]);
+
+const DATA = ((...dataNames) => {
+    /** @type {Record<string, any>} */
+    let result = {};
+    for (const name of dataNames) {
+        result[name] = readJSONFixture('interface-overview', name);
+    }
+    return result;
+})('EventTarget', 'XMLHttpRequestEventTarget');
```

Comment:
I still think that this code is clever instead of being clear and straightforward. Also DATA is a really undescriptive name. I'd suggest:

```
const FIXTURES = {
    EventTarget: readJSONFixture('interface-overview', 'EventTarget'),
    XMLHttpRequestEventTarget: readJSONFixture('interface-overview', 'XMLHttpRequestEventTarget')
};
```

That is shorter and easier to understand than in immediately-invoked anonymous function. Remember that this code base is largely created and maintained by technical writers, not software engineers, so I put a real premium on making the code straightforward.